scripts/integrationtest.sh

```diff
@@ -128,7 +128,7 @@ RECEIVED=$($BTC_CLIENT getreceivedbyaddress $PEG_OUT_ADDR)
 [[ "$RECEIVED" = "0.00000500" ]]
 
 # outgoing lightning
-INVOICE="$($LN2 invoice 100000 test test 1m | jq -r '.bolt11')"
+INVOICE="$($LN2 invoice 999999 test test | jq -r '.bolt11')"
```

Comment:
This effectively tests the change because the needs to make change because it doesn't have any millisats and this invoice wants 9 of them.